### ISSUE TITLE
.libPaths

### ISSUE THREAD

**Issue by mikelove** [genomicsclass/windows#2]
devtools won't install if the libPaths has a space in it, e.g. "C:/Program Files/..."

**Comment by yeredh** [comment]
I don't think this is an issue anymore. I came across this post in StackOverflow about this problem

http://stackoverflow.com/questions/9676894/installing-ggbiplot-from-github

He had spaces in libPaths and he couldn't install a package from GitHub. I checked, and my libPaths have spaces and I didn't run into this problem. To make sure I installed the package from the StackOverflow post

```
> library(devtools)
> install_github("vqv/ggbiplot")
Downloading github repo vqv/ggbiplot@master
Installing ggbiplot
Installing dependencies for ggbiplot:
ggplot2, plyr, scales
Installing packages into ‘C:/Users/APD/Documents/R/win-library/3.1’
(as ‘lib’ is unspecified)
also installing the dependencies ‘colorspace’, ‘gtable’, ‘reshape2’, ‘proto’, ‘dichromat’, ‘munsell’, ‘labeling’

* installing *source* package 'ggbiplot' ...
** R
** data
** preparing package for lazy loading
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded
*** arch - i386
*** arch - x64
* DONE (ggbiplot)
```

It worked! Note that the path for my library has spaces:

```
--library="C:/Users/APD/Documents/R/win-library/3.1"
```

**Comment by mikelove** [comment]
where are the spaces in your libPath?

here's an example of a student from yesterday who had a problem with a space:

https://courses.edx.org/courses/HarvardX/PH525.1x/1T2015/discussion/forum/i4x-HSPH-PH999x-course-2014_Spring_wk2/threads/54cd68e7af8bbdcae400008e

**Comment by mikelove** [comment]
ah, this might be because we are using install.packages("devtools"), not the devel version of devtools

**Comment by yeredh** [comment]
I think so. Because my path is

--library="C:/Users/APD/Documents/R/win-library/3.1"

and I didn't have any issues.

_Yered H. Pita-Juárez_
